gl2csv.py

```diff
@@ -1,13 +1,10 @@
 # -*- coding: utf-8 -*-
+import sys
 import argparse
 import csv
 import datetime
 import gitlab
-import pickle
 import iso8601
-import matplotlib.pyplot as plt
```

Comment:
Please don't remove. I understand can be a special case but I use it to generate statistic graphs. It would be OK for me to make this optional.